src/tests/GenreSelect.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { GenreSelect } from '../components/GenreSelect'
+import { render, screen } from '@testing-library/react'
+
+describe('GenreSelect', () => {
+    const genres = ['Action', 'Comedy', 'Drama']
+    const selectedGenre = genres[1]
+    const onGenreChange = jest.fn()
```

Comment:
The same as in `Search` tests - either clear the mocks after each test or use a setup function. Even though currently you call `onGenreChange` only once in the tests, in the future it may change and lead to unexpected results.